server/src/main/java/com/decathlon/ara/service/ProblemService.java

```diff
@@ -525,19 +525,20 @@ private ProblemWithAggregateDTO toProblemWithAggregate(Problem entity) {
      * Get all errors associated to the given problem id.
      *
      * @param projectId the ID of the project in which to work
-     * @param id        the id of the entity
+     * @param problemId        the id of the entity
      * @param pageable  the meta-data of the requested page
      * @return a page of the errors associated to the problem
      * @throws NotFoundException if the entity id does not exist
      */
     @Transactional(readOnly = true)
-    public Page<ErrorWithExecutedScenarioAndRunAndExecutionDTO> getProblemErrors(long projectId, long id, Pageable pageable) throws NotFoundException {
-        Problem problem = problemRepository.findByProjectIdAndId(projectId, id);
+    public Page<ErrorWithExecutedScenarioAndRunAndExecutionDTO> getProblemErrors(long projectId, long problemId, Pageable pageable) throws NotFoundException {
```

Comment:
Does this new behavior covered by a Unit test ?

Reply:
Done, but honestly I'm not a huge fan of commenting the source code: it should be self-explanatory.
The problem with comments is that when code changes they don't always follow. That is why it is important to work with a code that is easily understood by everyone working with it.

> "_A code never lies, comments sometimes do_" - Ron Jeffries



We can discuss it tomorrow if you want to ;)

Reply:
done